src/config.c

```diff
@@ -0,0 +1,48 @@
+#include "config.h"
+
+static volatile float micrometers_per_count = MICROMETERS_PER_COUNT;
+static volatile float wheels_separation = WHEELS_SEPARATION;
+static volatile struct control_constants control = {
+    KP_LINEAR,	KD_LINEAR,	KP_ANGULAR,      KD_ANGULAR,
+    KP_ANGULAR_FRONT, KI_ANGULAR_FRONT, KP_ANGULAR_SIDE, KI_ANGULAR_SIDE};
```

Comment:
I would use this syntax (check before using it, might be wrong, but there is something similar to specify the struct field name):

```
.kp_linear = KP_LINEAR, ...
```

Reply:
70cf7e59